pwnlib/util/sh_string.py

```diff
@@ -49,9 +49,9 @@
     Double Quotes
       Enclosing characters within double quotes preserves the literal meaning
       of all characters except dollarsign ($), backquote (`), and backslash
-      (\).  The backslash inside double quotes is historically weird, and
+      (\\).  The backslash inside double quotes is historically weird, and
       serves to quote only the following characters:
-            $ ` " \ <newline>.
+            $ ` " \\ <newline>.
```

Comment:
Can you elaborate on why this change is necessary?

Reply:
This should be done using `r"""` in line 3 instead, and leaving original content here and in other parts of this docstring.

Reply:
Added raw strings as suggested. Not sure why pyupgrade escaped them individually. I don't see any unicode characters in docstring affected by using raw literals.